client/coral-admin/src/components/ActionsMenuItem.js

```diff
@@ -3,7 +3,7 @@ import cn from 'classnames';
 import {MenuItem} from 'react-mdl';
 import PropTypes from 'prop-types';
 import styles from './ActionsMenu.css';
-import camelCase from 'lodash/camelcase';
+import camelCase from 'lodash.camelcase';
```

Comment:
Why the change?

Reply:
If it didnt work try `lodash/camelCase`

Reply:
Because it wasn't working on our CI
```
ModuleNotFoundError: Module not found: Error: Can't resolve 'lodash/camelcase' in '/home/ubuntu/talk/client/coral-admin/src/components'
    at factoryCallback (/home/ubuntu/talk/node_modules/webpack/lib/Compilation.js:260:39)
    at factory (/home/ubuntu/talk/node_modules/webpack/lib/NormalModuleFactory.js:247:20)
    at resolver (/home/ubuntu/talk/node_modules/webpack/lib/NormalModuleFactory.js:65:21)
    at asyncLib.parallel (/home/ubuntu/talk/node_modules/webpack/lib/NormalModuleFactory.js:138:21)
```

Reply:
did you try `lodash/camelCase`?

Reply:
Notice the case

Reply:
On my end both are working. I can give it a shot.

Reply:
Yeah some filesystems are case insensitive